pkg/env/scanner_v4_definitions_setting.go

```diff
@@ -0,0 +1,8 @@
+package env
+
+import "time"
+
+var (
+	// RepoMappingUpdateMaxInitialWait is the maximum wait time before the first repository mapping data
+	RepoMappingUpdateMaxInitialWait = registerDurationSetting("ROX_SCANNER_V4_CVSS_MAX_INITIAL_WAIT", 3*time.Minute)
```

Comment:
Can you make the variable name based on the environment variable? Example: `RepoMappingUpdateMaxInitialWait` => `ROX_MAPPING_UPDATE_MAX_INITIAL_WAIT`.